frontend-admin/src/pages/routes/routes-page.tsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
+import { useQuery } from 'react-query';
 import Card from '../../components/card/card';
 import './routes-page.scss';
 
+interface Route {
+  id: number;
+  name: string;
+}
+
+const fetchRoutes = async () => {
+  const response = await fetch('http://localhost:8000/routes');
```

Comment:
`http://localhost:8000/routes/` (Convention, less likely to be rejected by any id parser)

Reply:
yep, just hardcoding for demo

Reply:
fine ig, for the routes where we end with the routeID should we also include a trailing slash?

Reply:
Yes